examples/fluids/src/smartsim/sgs_dd_training.c

```diff
@@ -311,16 +318,20 @@ PetscErrorCode TSMonitor_SGS_DD_Training(TS ts, PetscInt step_num, PetscReal sol
     {
       const PetscScalar *training_data;
       PetscCall(VecGetArrayRead(TrainingData, &training_data));
+      PetscCall(PetscLogEventBegin(SmartRedis_Train, 0, 0, 0, 0));
       PetscSmartRedisCall(put_tensor(smartsim->client, array_key, array_key_len, (void *)training_data, sgs_dd_train->training_data_array_dims, 2,
-                                     SRTensorTypeDouble, SRMemLayoutContiguous));
+                                SRTensorTypeDouble, SRMemLayoutContiguous));
+      PetscCall(PetscLogEventEnd(SmartRedis_Train, 0, 0, 0, 0));
       PetscCall(VecRestoreArrayRead(TrainingData, &training_data));
     }
-    PetscCall(SmartRedisVerifyPutTensor(smartsim->client, array_key, array_key_len));
+    //PetscCall(SmartRedisVerifyPutTensor(smartsim->client, array_key, array_key_len));
 
     if (rank % smartsim->collocated_database_num_ranks == 0) {
       size_t   dim_2[1]      = {2};
       PetscInt step_array[2] = {step_num, step_num};
-      PetscSmartRedisCall(put_tensor(smartsim->client, "step", 4, step_array, dim_2, 1, SRTensorTypeInt32, SRMemLayoutContiguous));
+      PetscCall(PetscLogEventBegin(SmartRedis_Meta, 0, 0, 0, 0));
+      PetscSmartRedisCall(put_tensor(smartsim->client, "step", 4, step_array, dim_2, 1, SRTensorTypeInt64, SRMemLayoutContiguous));
```

Comment:
Same here in regards to the 32int to 64int change.

Reply:
Reverted to 32 bit Int, checked that training reads correct data